### ISSUE TITLE
[Bug] Piercing Howl (BB #716)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#716]
This issue was migrated from bitbucket.
**Original Reporter:** Flywithme
**Original Date:** 12.10.2014 14:18:06 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/716

<hr>

http://tbc.wowroad.info/?spell=12323 - not working properly. Don't detects stealth rouge.

http://www.wowwiki.com/Piercing_Howl - "_Ability also detects stealth just like Demoralizing Shout, Thunder Clap, and Shockwave._"

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Alex_Step
**Original Date:** 11.01.2015 14:36:29 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/716/#comment-14736335

<hr>

SpellAuras.cpp

```
#!c++
    // all applied/removed only at real aura add/remove
     if (!Real)
         return;
+       
+   switch (m_spellProto->Id)
+    {
+        case 12323: // Warrior: Piercing Howl
+             m_target->RemoveSpellsCausingAura(SPELL_AURA_MOD_STEALTH);
+             break;
+    }

     m_target->UpdateSpeed(MOVE_RUN, true);

```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 11.01.2015 14:58:39 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/716/#comment-14736606

<hr>

Do the other shouts detect stealth players?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Alex_Step
**Original Date:** 11.01.2015 15:02:09 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/716/#comment-14736640

<hr>

Me about it is not known exactly. It is also possible to output should: Warlock: Curse of Exhaustion, Warlock: Curse of Weakness. While I was looking for information to confirm.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 11.01.2015 16:20:14 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/716/#comment-14737490

<hr>

Fix #716 - thanks to Alex_Step

→ <<cset 02e7a681ec66>>
